File: javascript/codeeval/018_multiples_of_number.js
var fs  = require("fs");
var sm = function (inp) {
    inp = inp.split(',')
    t = parseInt(inp[0])
    f = parseInt(inp[1])
    m = 2
    p = f
    while (p < t) {
        p = f * m
        m++
    }
    return p
}
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        console.log(sm(line))
    }
});

File: python/codeeval/018_multiples_of_number.py
import sys

def sm(inp):
    inp = inp.split(',')
    t = int(inp[0])
    f = int(inp[1])
    m = 2
    p = f
    while (p < t):
        p = m * f
        m += 1
    return p

with open(sys.argv[1], 'r') as test_cases:
    for test in test_cases:
        print(sm(test))
